config/environments/development.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-    config.webpacker.check_yarn_integrity = true  # Settings specified here will take precedence over those in config/application.rb.
+    config.webpacker.check_yarn_integrity = false  # Settings specified here will take precedence over those in config/application.rb.
```

Comment:
Heheh, happy to merge once this is gone!!!

Reply:
@innatewonderer Now you can see PR comments too, i guess :D This PR is waiting on this boolean to go back to normal...

Reply:
On it